sources/packages/web/src/views/institution/OfferingsUpload.vue

```diff
@@ -0,0 +1,316 @@
+            <Column
+              header="End date"
+              field="endDateFormatted"
+              bodyClass="text-no-wrap"
+            ></Column>
+            <Column header="Status"
+              ><template #body="slotProps">
+                <status-chip-offering
+                  v-if="slotProps.data.offeringStatus"
+                  :status="slotProps.data.offeringStatus"
+                /> </template
+            ></Column>
+            <Column header="Validations"
+              ><template #body="slotProps">
+                <div
+                  class="alert alert-danger"
+                  v-if="slotProps.data.errors.length"
+                >
+                  <ul class="m-2">
+                    <li v-for="error in slotProps.data.errors" :key="error">
+                      {{ error }}
+                    </li>
+                  </ul>
+                </div>
+                <div
+                  class="alert alert-warning"
+                  v-if="slotProps.data.warnings.length"
+                >
+                  <ul class="m-2">
+                    <li
+                      v-for="warning in slotProps.data.warnings"
+                      :key="warning"
+                    >
+                      {{ warning.warningMessage }}
+                    </li>
+                  </ul>
+                </div>
+              </template></Column
+            >
+          </DataTable>
+        </content-group>
+      </content-group>
+    </content-group>
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { ref, computed } from "vue";
+import {
+  OfferingsUploadBulkInsert,
+  DEFAULT_PAGE_LIMIT,
+  PAGINATION_LIST,
+  OfferingStatus,
+  BannerTypes,
+  VForm,
+  FileInputFile,
+} from "@/types";
+import { EducationProgramOfferingService } from "@/services/EducationProgramOfferingService";
+import StatusChipOffering from "@/components/generic/StatusChipOffering.vue";
+import { useSnackBar } from "@/composables";
+import { FileUploadProgressEventArgs } from "@/services/http/common/FileUploadProgressEvent";
+
+const ACCEPTED_FILE_TYPE = "text/csv";
+const MAX_OFFERING_UPLOAD_SIZE = 4194304;
+
+export default {
+  components: {
+    StatusChipOffering,
+  },
+  setup() {
+    const snackBar = useSnackBar();
+    const loading = ref(false);
+    // Only one will be used but the component allows multiple.
+    const offeringFiles = ref([] as FileInputFile[]);
+    // Possible errors and warnings received upon file upload.
+    const validationResults = ref([] as OfferingsUploadBulkInsert[]);
+    const uploadForm = ref({} as VForm);
+    const uploadProgress = ref({} as FileUploadProgressEventArgs);
+    const showValidationSummary = ref(false);
+    // Workaround to reset the file upload component to its original state.
+    // It is apparently a vuetify beta issue. It can be removed once there is a
+    // better way to force the component to reset its state.
+    const csvFileUploadKey = ref(0);
```

Comment:
what is the issue here?

Reply:
@andrewsignori-aot is that the error thrown at $form.reset()?

Reply:
There is no error actually thrown but the file upload is not reset property. The expectation is to set the model to `null` and it should go back to the original state or call the `form.reset()` and the form goes back to its original state, but the v-file-upload is not doing it properly.

Reply:
oh..okay